ocaml/xapi/xapi_pci_helpers.ml

```diff
@@ -28,8 +28,17 @@ type pci = {
   subsystem_vendor: pci_property option;
   subsystem_device: pci_property option;
   related: string list;
+  driver_name: string option;
 }
 
+let get_driver_name address =
+    try
+      let driver_path = Unix.readlink (Printf.sprintf "/sys/bus/pci/devices/%s/driver" address) in
```

Comment:
We need to do this because `pciutils` does not know about it (see recent relevant commit: https://github.com/pciutils/pciutils/commit/11339c0d3d0c9064717983fa283bafcdc5298fe7)

Reply:
Let me clarify the comment. We cannot use `libpci` (part of `pciutils`) to read the driver name, because it does not have the functionality. In fact, `lspci` (part of `pciutils` as well) does it exactly as we are doing here (see the commit linked above). So I think this is the way to go to read the driver name